support/linux/provision.sh

```diff
@@ -1,43 +1,11 @@
 #!/bin/bash
-
-export APP_HOSTNAME=localhost:3000
-export GITHUB_API_URL=https://api.github.com
-export GITHUB_WEB_URL=https://github.com
-export GITHUB_CLIENT_ID=Iv1.732260b62f84db15
-export GITHUB_CLIENT_SECRET=fc7654ed8c65ccfe014cd339a55e3538f935027a
-export WORKER_AUTH_TOKEN=fc7654ed8c65ccfe014cd339a55e3538f935027a
-export GITHUB_ADMIN_TEAM=1995301
-export GITHUB_WORKER_TEAM=2555389
-export GITHUB_WEBHOOK_SECRET=58d4afaf5e5617ab0f8c39e505605e78a054d003
-export GITHUB_APP_ID=5629
-pushd /src
-cp components/hab/install.sh /tmp/
-sh support/linux/install_dev_0_ubuntu_latest.sh
-sh support/linux/install_dev_9_linux.sh
+set -eux
+
+sudo sh /src/components/hab/install.sh
```

Comment:
There's nothing here or in the directions that create a `/src` link. It exists in the studio, but not outside, so either something could create it, or this could be modified to not need it.

Reply:
Something does create it - the Vagrantfile ;)

I'll modify it to use curl|bash